tensorflow/lite/micro/kernels/if.cc

```diff
@@ -65,9 +65,7 @@ TfLiteStatus Prepare(TfLiteContext* context, TfLiteNode* node) {
   // GetExecutionPlan from TfLiteContext. On TFLM this method returns a
   // MicroGraph.
   // TODO(b/188226309): Design a cleaner way to get a graph from kernel context.
-  MicroGraph* graph_info;
-  context->GetExecutionPlan(context,
-                            reinterpret_cast<TfLiteIntArray**>(&graph_info));
+  MicroGraph* graph_info = micro::GetMicroGraph(context);
```

Comment:
let's remove the TODO above and close that bug in favor of whatever bug you have made for the current PR.

Reply:
Done